vendor/modules.txt

```diff
@@ -55,11 +55,6 @@ github.com/aws/aws-sdk-go/service/sts/stsiface
 github.com/beorn7/perks/quantile
 # github.com/cespare/xxhash/v2 v2.1.1
 github.com/cespare/xxhash/v2
-# github.com/coreos/prometheus-operator v0.38.1-0.20200424145508-7e176fda06cc => github.com/coreos/prometheus-operator v0.38.1-0.20200424145508-7e176fda06cc
```

Comment:
Please add the command used to update the vendor directory.

Reply:
done